components/automate-ui/src/app/components/process-progress-bar/process-progress-bar.component.ts

```diff
@@ -66,4 +66,13 @@ export class ProcessProgressBarComponent implements OnInit, OnDestroy {
   public cancelApplyStop(): void {
     this.closeConfirmApplyStopModal();
   }
+
+  public displayPercent(): string {
+    if (this.percentageComplete === 0) {
+      return 'Preparing...';
+    } else if (this.percentageComplete === 100) {
```

Comment:
Seems like this should match the changes above exactly:
```suggestion
    if (Math.floor(this.percentageComplete) === 0) {
      return 'Preparing...';
    } else if (Math.ceil(this.percentageComplete) === 100) {
```